### ISSUE TITLE
serialize / deserialize?

### ISSUE THREAD

**Issue by RGBboy** [Matt-Esch/virtual-dom#130]
I'm wondering if there is a way to serialize a vtree to send over the wire and deserialize it on the other side?

**Comment by nolanlawson** [comment]
For the record, I wrote a JSON converter: https://github.com/nolanlawson/vdom-as-json

**Comment by RangerMauve** [comment]
Any reason for not going with a binary serialization other than easy of implementation?

**Comment by nolanlawson** [comment]
Yeah for perf reasons I ultimately went with yet another approach: https://github.com/nolanlawson/vdom-serialized-patch
